backend/main/services/empi/empi_service.py

```diff
@@ -809,9 +810,27 @@ def _update_person_records(
             )
             total_record_updated_count += record_updated_count
 
-        if total_record_updated_count != len(add_action_partials):
+        if review_action_partials:
+            self.logger.info(
+                f"Updating {len(review_action_partials)} PersonRecords as reviewed"
+            )
+            for action in review_action_partials:
+                record_updated_count = PersonRecord.objects.filter(
+                    id=action["person_record_id"]
+                ).update(
+                    matched_or_reviewed=match_event.created,
+                )
+                total_record_updated_count += record_updated_count
```

Comment:
```suggestion
                total_record_updated_count += record_updated_count

```

This threw me off a bit when reading since python doesn't have braces around if blocks